<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ski Resort Visualizer</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>460</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QFrame" name="GpsFrame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS Coordinate X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS Coordinate Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="coord_x">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="calcSpeedButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate Speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="coord_y">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="enableGpsCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>190</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable GPS output</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="TotalDistanceFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total distance</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalDistanceLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="calcDistanceButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>190</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="highestSpeedButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Show me the highest speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="videoSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>521</width>
      <height>19</height>
     </rect>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/stop.ico</normaloff>:/images/stop.ico</iconset>
    </property>
   </widget>
   <widget class="QFrame" name="RegionFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Point #1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Point #2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showRegionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>190</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Region</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="point1x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="point1y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="point2y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="point2x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Visualizer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
